### PR TITLE
Feature/navbar

### PR DESCRIPTION
**Pull request by schittlur**

### CODE REVIEW

**Review by mattdrayer — APPROVED** [review]
Unfortunately my front-end skills are inferior to my back-end skills, so this PR review is only slightly more than a rubber stamp ✅

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    <section className="col span-1-of-6">
+                        <div>
+                            <h3 className={classes.text}>DOD Digital Learning Portal</h3>
+                        </div>
+                    </section>
+                    <section className="col span-2-of-6">
+                        <div className="row">
+                            <h4 className={classes.text}>Links</h4>
+                            <ul className="col span-1-of-3">
+                                <li className={classes.listLinks}><a href="https://dodcio.defense.gov/" target="_blank" className={classes.links}>Home</a></li>
```
Comment (jdiouf) [inline src/components/Footer/Footer.js:20]: I get a warning for each link element. I think we need rel="noreferrer" for all links.

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            </form>
+            
+            <NavLink to='/signIn' className={classes.signIn}>Sign in</NavLink>
+            </div>         
+        </div>
+    )
+    
+}
+
+export default Header;
```
Comment (jdiouf) [inline src/components/Header/Header.js:43]: Need a new line character at the end

File: src/components/Header/Header.module.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    width: 197px;
+    display: inline;
+}
+
+.searchImage{
+    height: 39px;
+    padding-bottom: 10px;
+    display: inline;
+    vertical-align: text-top;
+}
```
Comment (jdiouf) [inline src/components/Header/Header.module.css:97]: Need new line character here

File: src/components/Navigation/Navbar/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+            </ul>
+            <ul className={classes.navBar}>
+                <li className={classes.navItem}></li>
+            </ul>
+        </>
+
+    )
+}
+
+export default navbar;
```
Comment (jdiouf) [inline src/components/Navigation/Navbar/Navbar.js:25]: Need new line character at the end.

**Review by jdiouf — APPROVED** [review]

**Comment by jdiouf** [comment]
Changes have been integrated with feature/landing-page branch.

**Pull request closed without merging**